docs/AA--Javascript-in-EE/components/form/select/datetime-select.md

```diff
@@ -0,0 +1,152 @@
+# `<DatetimeSelect />`
+
+DatetimeSelect is a component that exposed the [`react-select`](https://deploy-preview-2289--react-select.netlify.com/home) component for displaying provided EE Datetime model data for selection. It is a wrapper surrounding the [`<ModelEnhancedSelect />`](model-select.md) higher order component.
+
+## Shape
```

Comment:
"Shape" seems to be common vernacular for referring to the structure of an element.  The purpose of this section is to simply highlight the default structure (or shape) of the component.  I don't think "Usage" is accurate here because there's more involved with "using" the component then simply copying and pasting this code.  The suggested title for the current "Usage Overview" (Examples) doesn't seem accurate because that section doesn't really have any full examples for usage (but merely gives some usage instruction imo).

I personally am fine with the current headings as I feel they accurate describe the contents.  I'm open to using something else to replace "Shape" if you feel that's not helpful.  What about `Structure` or `Component` or `Component Structure` or even just DROP shape as a heading and leave it's content as part of the initial description?

Reply:
I was trying to bring our documentation more in line with what WordPress core is doing where they appear to either use "Usage" or "Example" prior to showing the component and its properties. I know I have also seen them use "Overview" but it wasn't an actual component since I've gone through all of those looking for a precedent to refer to, but don't feel like going through every single one of their other docs to find what I saw.

I think "Shape" is too odd sounding even if it is an accurate descriptor, and would prefer something more generic.

IMHO the following general schema makes the most sense:

**Overview** 
the initial introduction to the component without getting into any nitty gritty details

**Props**
list of the component's props and their details

**Usage** 
the nitty gritty details including setup, configuration, possible code examples, and any other information that a dev may need to know

I won't say anymore and leave the final decision up to you as I don't want to bikeshed this anymore than we have.